pdns/mastercommunicator.cc

```diff
@@ -93,7 +95,7 @@ void CommunicatorClass::queueNotifyDomain(const DNSName &domain, UeberBackend *B
     }
   }
 
-  if (!hasQueuedItem)
+  if (!hasQueuedItem && d_onlyNotify.size())
```

Comment:
How about a startup warning that empty `only-notify` prevents all notifications?